lunches.py

```diff
@@ -390,6 +390,45 @@ def caesar(dom):
 def morgans(dom):
     yield from menicka_parser(dom)
 
+@restaurant("U Mořice", "https://www.menicka.cz/api/iframe/?id=5299", Location.Olomouc)
+def moric(dom):
+    yield from menicka_parser(dom)
+
+def slepico_parser(dom):
+    current_day = datetime.datetime.now().strftime("%-d.%-m.%Y")
+    slepico_regex = re.compile("^(\d\s?[,.]\s?\d\s?l?\s)?(?P<soup>[^+]+)(\s+\+\s+)(\d+g)?(?P<lunch>.*)$")
```

Comment:
using `\s*` may be safer - means zero or more spaces (will less likely break if they omit the space or add more of them)
You dont need to create group for `+` surrounding - `\s*\+\s*`
Maybe also add spaces around "weight" - `\d+\s*g\s*`?